lib/routing.js

```diff
@@ -62,7 +65,20 @@ export const deserializeState = serializedState => {
   return JSON.parse(decodeURIComponent(stateString))
 }
 
-const getQueryStringObject = query => {
+export const getRouteState = location => {
+  const { query, pathname } = url.parse(location, true)
+  const state = getQueryStringState(query)
```

Comment:
Also if `getQueryStringState` is no longer used outside of this function, make sure it's not exported :+1: